test/sdf/model_top_level_frame.sdf

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
+<sdf version="1.8">
+  <model name="M">
+    <pose relative_to="some_frame"/>
```

Comment:
Since this is an invalid example, can you add a comment here stating so? Just want to avoid any confusion for anyone reading this..

Reply:
Good catch, didn't notice all the invalid model files have the word `invalid` in their names too, just made the changes, https://github.com/ignitionrobotics/sdformat/pull/636/commits/e1e9b911936d63f0ab7d407600a2d710c54511fc

Reply:
Thanks!